api/extension-guides/notebook.md

```diff
@@ -278,25 +285,26 @@ Renderers are declared for a set of mimetypes by contributing to the `contribute
 
 Output renderers are always rendered in a single `iframe`, separate from the rest of VS Code's UI, to ensure they don't accidentally interfere or cause slowdowns in VS Code. The contribution refers to an "entrypoint" script, which is loaded into the notebook's `iframe` right before any output needs to be rendered. Your entrypoint needs to be a single file, which you can write yourself, or use a bundler like Webpack, Rollup, or Parcel to create.
 
-When it's loaded, your entrypoint script should immediately call `acquireNotebookRendererApi()` with your renderer ID, and start listening to notebook output events. For example, this will put all your GitHub issue data as JSON into the cell output:
+When it's loaded, your entrypoint script should immediately call `ActivationFunction` from `vscode-notebook-renderer` to render your UI once VS Code is ready to activate and render your renderer. For example, this will put all your GitHub issue data as JSON into the cell output:
```

Comment:
The renderer is no longer calling anything, it's exporting the activation function.